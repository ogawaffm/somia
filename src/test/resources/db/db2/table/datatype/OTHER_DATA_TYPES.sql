DROP TABLE IF EXISTS TEST.OTHER_DATA_TYPES
/
CREATE TABLE TEST.OTHER_DATA_TYPES
(
    BOOLEAN_COL BOOLEAN,
    XML_COL     XML

)