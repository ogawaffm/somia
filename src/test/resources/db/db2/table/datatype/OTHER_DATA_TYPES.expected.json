{
  "name": "OTHER_DATA_TYPES",
  "catalogName": "",
  "schemaName": "TEST",
  "typeName": "TABLE",
  "comment": "",
  "columns": [
    {
      "name": "BOOLEAN_COL",
      "type": {
        "name": "BOOLEAN",
        "typeNumber": 16,
        "className": "java.lang.Boolean",
        "precision": 1,
        "scale": 0,
        "isNullable": true,
        "isAutoIncrement": false,
        "numericPrecisionRadix": 0,
        "isCaseSensitive": false,
        "isSearchable": true,
        "isSigned": false,
        "isCurrency": false,
        "columnDisplaySize": 5,
        "baseType": null
      },
      "comment": "",
      "collation": "",
      "defaultDefinition": ""
    },
    {
      "name": "XML_COL",
      "type": {
        "name": "XML",
        "typeNumber": 1111,
        "className": "java.lang.Object",
        "precision": 0,
        "scale": 0,
        "isNullable": true,
        "isAutoIncrement": false,
        "numericPrecisionRadix": 0,
        "isCaseSensitive": false,
        "isSearchable": true,
        "isSigned": false,
        "isCurrency": false,
        "columnDisplaySize": 0,
        "baseType": null
      },
      "comment": "",
      "collation": "",
      "defaultDefinition": ""
    }
  ],
  "indexes": [
    {
      "name": "",
      "columns": [],
      "isUnique": false,
      "comment": "",
      "qualifierName": "",
      "isClustered": false,
      "isHashed": false,
      "isStatistic": true,
      "whereClause": "",
      "cardinality": 0,
      "pages": 0
    }
  ],
  "collation": "",
  "primaryKey": null,
  "bestRowIdentifier": null
}
